src/accelerator/ogl/image/image_kernel.cpp

```diff
@@ -393,9 +395,9 @@ struct image_kernel::impl
                 GL(glEnableVertexAttribArray(tex_loc));
 
                 GL(glVertexAttribPointer(vtx_loc, 2, GL_DOUBLE, GL_FALSE, stride, nullptr));
-                GL(glVertexAttribPointer(tex_loc, 4, GL_DOUBLE, GL_FALSE, stride, nullptr));
+                GL(glVertexAttribPointer(tex_loc, 4, GL_DOUBLE, GL_FALSE, stride, (GLvoid*)(2 * sizeof(GLdouble))));
 
-                for (int i = 0; i < coords.size(); i += 3) {
+                for (int i = 0; i < coords_triangles.size(); i += 3) {
```

Comment:
quads or triangles?

Reply:
frame_geometry is back to defining quads, as it makes the transform maths simpler as that is what it is already expecting.
but opengl doesnt support drawing quads anymore, so I am converting the quads to triangles after any transforms are calculated